gc_registry/certificate/services.py

```diff
@@ -592,7 +594,7 @@ def query_certificate_bundles(
     # certificates
     stmt: SelectOfScalar = select(GranularCertificateBundle).where(
         GranularCertificateBundle.account_id == certificate_query.source_id,
-        ~GranularCertificateBundle.is_deleted,
```

Comment:
Does the tilde not work?

Reply:
It works but throws a deprecation warning on my linter so changed it to something that stopped complaining

Reply:
Ah no *that* is because sqlalchemy needs a non-standard treatment of boolean filtering that Ruff doesn't like :p